plugins/GSdx/GSDevice.h

```diff
@@ -25,6 +25,7 @@
 #include "GSTexture.h"
 #include "GSVertex.h"
 #include "GSAlignedClass.h"
+#include "GSOsdManager.h"
 
 enum ShaderConvert {
```

Comment:
Could you extand this enum and uses it instead of 18/19 value for the program object